openfe/protocols/openmm_afe/afe_settings.py

```diff
@@ -0,0 +1,373 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+"""Settings class for equilibrium AFE Protocols using OpenMM + OpenMMTools
+
+This module implements the necessary settings necessary to run absolute free
+energies using :class:`openfe.protocols.openmm_abfe.equil_abfe_methods.py`
+
+
+TODO
+----
+* Add support for restraints
+* Improve this docstring by adding an example use case.
+
+"""
+from typing import Optional
+from pydantic import validator
+from openff.units import unit
+from gufe import settings
+
+
```

Comment:
@richardjgowers this is pretty much what I'm proposing as to how the RBFE settings should look.

Once we get those settings updated I can go ahead and switch to using a similar parameterisation method as what's used in this protocol to make sure we can have arbitrary small molecules.